### PR TITLE
September improvements by @backslash47

### PR DESCRIPTION
**Pull request by danfickle**
Thanks for the awesome work @backslash47

### CODE REVIEW

**Comment by danfickle** [comment]
Hi @backslash47 

The symbol ```showTextWithPositioning``` can not be found on the PDF-BOX content stream. I guess this is using a private version of PDF-BOX? If you could upload the code somewhere, even as a gist, or point me to it, I could file an issue with PDF-BOX to get this method included.

Thanks,
Daniel.

**Comment by backslash47** [comment]
Hello,
I have it here: https://github.com/backslash47/pdfbox/commit/3c528295b16445e58dc9fe895f78384221452be2

I did not have much time to work it into PDFBOX repo. If you can propose it to pdfbox maintainers, it would be great.

File: openhtmltopdf-pdfbox/src/main/java/com/openhtmltopdf/pdfboxout/PdfBoxRenderer.java [hunk 1]
```diff
@@ -169,11 +179,12 @@ public PdfBoxRenderer(float dotsPerPoint, int dotsPerPixel, boolean useSubsets,
     PdfBoxRenderer(BaseDocument doc, UnicodeImplementation unicode, 
             HttpStreamFactory httpStreamFactory, 
             OutputStream os, FSUriResolver resolver, FSCache cache, SVGDrawer svgImpl,
-            PageDimensions pageSize, float pdfVersion, String replacementText, boolean testMode, FSObjectDrawerFactory objectDrawerFactory) {
+            PageDimensions pageSize, float pdfVersion, String pdfAConformance, byte[] colorProfile, String replacementText, boolean testMode, FSObjectDrawerFactory objectDrawerFactory) {
```
Comment (schmitch) [inline openhtmltopdf-pdfbox/src/main/java/com/openhtmltopdf/pdfboxout/PdfBoxRenderer.java:182]: I'm not sure if it is a good idea to pass the pdfAConformance as a string.

They actually barley change over the years.

**Comment by danfickle** [comment]
We're now waiting on [PDFBOX-3992](https://issues.apache.org/jira/browse/PDFBOX-3992) for the show text with positioning operator (TJ).

**Comment by danfickle** [comment]
Should be finished with this one.

**Pull request closed without merging**